app/javascript/components/search-bar/index.jsx

```diff
@@ -23,7 +23,7 @@ const SearchBar = ({ searchText, advancedSearch, action }) => {
 
   /** Function to filter the results through form submit. */
   const onSearch = () => {
-    http.post({ url: action });
+    http.post(action, null, {headers: {}, skipJsonParsing: true});
```

Comment:
PR looks good now, just one small fix. There is a lint error on this line, it should have spaces after the { and before the }. As so: 
`http.post(action, null, { headers: {}, skipJsonParsing: true });`

Reply:
fixed, thanks!